esphome/components/ethernet/ethernet_component.h

```diff
@@ -58,7 +58,7 @@ class EthernetComponent : public Component {
   void set_clk_mode(emac_rmii_clock_mode_t clk_mode, emac_rmii_clock_gpio_t clk_gpio);
   void set_manual_ip(const ManualIP &manual_ip);
 
-  network::IPAddress get_ip_address();
+  network::IPAddresses get_ip_address();
```

Comment:
I feel like if you are changing the functionality to return a list, the name of the function should also change...
```suggestion
  network::IPAddresses get_ip_addresses();
```

Reply:
Good suggestion! Should I also update `WiFiComponent::wifi_sta_ip` to `WiFiComponent::wifi_sta_ips`?